packages/babel-code-frame/src/index.js

```diff
@@ -24,14 +23,81 @@ let defs = {
  */
 
 const NEWLINE = /\r\n|[\n\r\u2028\u2029]/;
+const KEYWORDS = {
+  "abstract": true,
+  "await": true,
+  "boolean": true,
+  "break": true,
+  "byte": true,
+  "case": true,
+  "catch": true,
+  "char": true,
+  "class": true,
+  "const": true,
+  "continue": true,
+  "debugger": true,
+  "default": true,
+  "delete": true,
+  "do": true,
+  "double": true,
+  "else": true,
+  "enum": true,
+  "export": true,
+  "extends": true,
+  "false": true,
+  "final": true,
+  "finally": true,
+  "float": true,
+  "for": true,
+  "function": true,
+  "goto": true,
+  "if": true,
+  "implements": true,
+  "import": true,
+  "in": true,
+  "instanceof": true,
+  "int": true,
+  "interface": true,
+  "let": true,
+  "long": true,
+  "native": true,
+  "new": true,
+  "null": true,
+  "package": true,
+  "private": true,
+  "protected": true,
+  "public": true,
+  "return": true,
+  "short": true,
+  "static": true,
+  "super": true,
+  "switch": true,
+  "synchronized": true,
+  "this": true,
+  "throw": true,
+  "transient": true,
+  "true": true,
+  "try": true,
+  "typeof": true,
+  "var": true,
+  "void": true,
+  "volatile": true,
+  "while": true,
+  "with": true,
+  "yield": true
+};
```

Comment:
I don't have a strong opinion about using or not using esutils, but I think it would be nice to have a single source of truth for keywords throughout all the babel packages.